pyproject.toml

```diff
@@ -34,7 +34,7 @@ docutils = ">=0.16"
 flask = ">=2.0.1"
 fedora-messaging = ">=2.1.0"
 "moksha.common" = "^1.2.5"
-pygal = ">=2.4.0"
+pygal = ">=3.0.0"
```

Comment:
Do we use features in pygal 3 that are not in pygal 2.4 ?

Reply:
> Do we use features in pygal 3 that are not in pygal 2.4 ?

Not really. Changing it back. It was because in koji I saw the version 3.0.0 of pygal  in f36 and f37.